client/checkout/classic/event-handlers.js

```diff
@@ -92,6 +92,22 @@ jQuery( function ( $ ) {
 		return processPaymentIfNotUsingSavedMethod( $( this ) );
 	} );
 
+	$( document.body ).on( 'payment_method_selected', function () {
+		// Get selected payment method using WooCommerce's approach, see /legacy/js/frontend/checkout.js
+		const selectedPaymentMethod = $(
+			'.woocommerce-checkout input[name="payment_method"]:checked'
```

Comment:
We're approaching things the same way in WooCommerce [`checkout.js`](https://github.com/Automattic/woocommerce/blob/security/plugins/woocommerce/client/legacy/js/frontend/checkout.js#L120)